plugins/cloudbuild/src/components/WorkflowRunDetails/WorkflowRunDetails.tsx

```diff
@@ -86,10 +86,12 @@ export const WorkflowRunDetails = ({ entity }: { entity: Entity }) => {
 
   return (
     <div className={classes.root}>
-      <Breadcrumbs aria-label="breadcrumb">
-        <Link to="..">Workflow runs</Link>
-        <Typography>Workflow run details</Typography>
-      </Breadcrumbs>
+      <Box mb={3}>
+        <Breadcrumbs aria-label="breadcrumb">
+          <Link to="..">Workflow runs</Link>
+          <Typography>Workflow run details</Typography>
```

Comment:
I know it was here before, but I'm interested that this uses the same terminology as GitHub Actions. I wonder if this was a copy and paste to get this plugin out there.

https://cloud.google.com/build/docs/overview

It seems like GCP calls these Builds and Build Results but I could be wrong? do we know anyone who knows more about it? Spotify is a big GCP shop, any ideas?

Reply:
<img width="1436" alt="Capture d’écran 2021-02-25 à 20 37 08" src="https://user-images.githubusercontent.com/4517991/109207312-47a36100-77a9-11eb-9e2d-18f1539060d1.png">
<img width="1437" alt="Capture d’écran 2021-02-25 à 20 37 13" src="https://user-images.githubusercontent.com/4517991/109207320-4a9e5180-77a9-11eb-8c57-120f476fd378.png">

I suggest using "Build history" and "Build details" as suggested in the screenshots I took from a video demo. I can easily include the change in this PR if you're ok with it

Reply:
@ebarriosjr you may have been one of the early developers on this one, might you know?

Reply:
Oh awesome yeah @wrousseau nice one! Yeah I just figure you're in there, it's a small change, may as well avoid another PR loop. May be best to add a second changeset just for CloudBuild and say the terminology has updated (so leave the one you have, and add one more) but if you don't mind squeezing it in I think a good incremental alignment!!!

Reply:
@adamdmharvey done :)